packages/react/src/wallet/ConnectWallet/screens/Metamask/MetamaskGetStarted.tsx

```diff
@@ -1,10 +1,10 @@
-import { MetamaskIcon } from "../../icons/MetamaskIcon";
+import { MetamaskWallet } from "../../../wallets";
 import { GetStartedScreen } from "../GetStartedScreen";
 
 export const MetamaskGetStarted: React.FC<{ onBack: () => void }> = (props) => {
   return (
     <GetStartedScreen
-      WalletIcon={MetamaskIcon}
+      walletIconURL={MetamaskWallet.meta.iconURL}
       walletName="Metamask"
       chromeExtensionLink="https://chrome.google.com/webstore/detail/metamask/nkbihfbeogaeaoehlefnkodbefgpgknn"
       googlePlayStoreLink="https://play.google.com/store/apps/details?id=io.metamask"
```

Comment:
and these, these feel meta-y?

Reply:
Good catch!, rainbow-kit also puts all these in meta. we can do the same

<img width="1174" alt="image" src="https://user-images.githubusercontent.com/22043396/225707239-7aac37c7-04ac-4cb7-ae82-7645901f856f.png">